dts/bindings/base/base.yaml

```
@@ -17,78 +26,169 @@ properties:
   compatible:
     type: string-array
     required: true
-    description: compatible strings
+    description: |
+      This property is a list of strings that essentially define what
+      type of hardware or other resource this devicetree node
+      represents. Each device driver checks for specific compatible
+      property values to find the devicetree nodes that represent
+      resources that the driver should manage.
+
+      The recommended format is "vendor,device", The "vendor" part is
+      an abbreviated name of the vendor. The "device" is usually from
+      the datasheet.
+
+      The compatible property can have multiple values. Additional values are
+      useful when the device is a specific instance of a more general family, to
+      allow the system to match from most- to least-specific device drivers.
+
+      For details, see "2.3.1 compatible" in Devicetree Specification v0.4.
 
   reg:
     type: array
-    description: register space
+    description: |
+      Information used to address the device. The value is specific to
+      the device (i.e. is different depending on the compatible
+      property).
+
+      The "reg" property is a sequence of (address, length) pairs.
```

This is not always true. It depends on the parent `#size-cells` value. Perhaps change it to:

```suggestion
      The "reg" property is typically a sequence of (address, length) pairs.
```

---

applied verbatim